internal/api/api.go

```diff
@@ -14,6 +16,8 @@ import (
 type ApiClient interface {
 	GetDefaultOrgId() (orgID string, err error)
 	GetOrgIdFromSlug(slugName string) (string, error)
+	SetUrl(url string)
```

Comment:
Suggestion: Since it seems rarely reasonable to set only one Url or client, why not combine them in a single method eg. Init(url, client). This could be called as well in NewApi().

Reply:
updated!